COLLABORATOR_GUIDE.md

```diff
@@ -229,6 +229,10 @@ is designed to allow validation of changes to the copy of V8 in the Node.js
 tree by running the standard V8 tests. It should be run whenever the
 level of V8 within Node.js is updated or new patches are floated on V8.
 
+* [`node-test-commit-custom-suites`](https://ci.nodejs.org/job/node-test-commit-custom-suites/)
+runs tests that are in the `test/internet` directory. Those are not run by the
```

Comment:
Not exactly... how about:

[`node-test-commit-custom-suites`](https://ci.nodejs.org/job/node-test-commit-custom-suites/) runs tests in user-specified directories with parameters provided by the user as well. Currently used on all `node-test-commit` jobs to run the `default` test suite with the `--worker` flag.

Reply:
@maclover7 Updated. PTAL